### PR TITLE
Update supported platforms

### PR DESCRIPTION
**Pull request by HanaGemela**

### CODE REVIEW

File: docs/modules/ROOT/pages/installing.adoc [hunk 1]
```diff
@@ -17,12 +17,12 @@ You will also find links to source code archives and older versions on the downl
...
 
 * Windows 7+
-* Mac OS X 10.7+ (*64-bit only*)
-* CentOS 6 & 7 (64-bit only)
-* Debian 8.0 & 9.0
-* Fedora 25 & 26 & 27
-* Ubuntu 16.04 & 17.04 & 17.10
-* openSUSE Leap 42.2 & 42.3
+* Mac OS X 10.10+ (*64-bit only*)
+* CentOS 7 (64-bit only) & 8
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/installing.adoc]: With the 2.6.1, C6 is still supported?

Comment (HanaGemela) [reply docs/modules/ROOT/pages/installing.adoc]: Yes, support dropped after the release https://central.owncloud.org/t/desktop-client-supported-platforms-updated/24165

Comment (michaelstingl) [reply docs/modules/ROOT/pages/installing.adoc]: No, not dropped after release of the 2.6.1. The release **after** the 2.6.1 won't support it. As long as the 2.6.1 is the latest stable, official release, C6 should be supported. The day we have a 2.6.2 or a 2.7.0, C6 is no longer supported.

File: docs/modules/ROOT/pages/installing.adoc [hunk 2]
```diff
@@ -17,12 +17,12 @@ You will also find links to source code archives and older versions on the downl
...
 
 * Windows 7+
-* Mac OS X 10.7+ (*64-bit only*)
-* CentOS 6 & 7 (64-bit only)
-* Debian 8.0 & 9.0
-* Fedora 25 & 26 & 27
-* Ubuntu 16.04 & 17.04 & 17.10
-* openSUSE Leap 42.2 & 42.3
+* Mac OS X 10.10+ (64-bit only)
+* CentOS 6 & 7 (64-bit only) & 8
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/installing.adoc]:
```suggestion
* CentOS 6 & 7 & 8
```

All Linux only for 64-bit

File: docs/modules/ROOT/pages/installing.adoc [hunk 3]
```diff
@@ -17,12 +17,12 @@ You will also find links to source code archives and older versions on the downl
...
+* Debian 8.0 & 9.0 & 10
+* Fedora 27 & 28 & 29 & 30 & 31
+* Ubuntu 18.04 & 19.04 & 19.10
+* openSUSE Leap 42.2 & 42.3 & 15.0 & 15.1 & 15.02
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/installing.adoc]:
```suggestion
* openSUSE Leap 42.2 & 42.3 & 15.0 & 15.1 & 15.2
```

**Pull request merged**